pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -57,16 +53,11 @@ func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfigura
 		return docker.AuthConfiguration{}, false
 	}
 
-	var cfg credentialprovider.DockerConfig
```

Comment:
agreed, if we delete 44-46 lines, we should delete the line `len(pathForAuthType) > 0 `